src/main/scala/mesosphere/marathon/state/Volume.scala

```diff
@@ -112,8 +112,12 @@ case class PersistentVolume(
 
 object PersistentVolume {
   import org.apache.mesos.Protos.Volume.Mode
+
+  val NoSlashesPattern = """^[^/]+$""".r
```

Comment:
The pattern also matches prevents empty strings (which is wanted), but the name doesn't reflect that. I'd be ok with a generic `ContainerPathPattern`